eslint.json

```diff
@@ -70,6 +70,7 @@
         "exceptions": ["*"]
       }
     }],
-    "strict": ["error", "global"]
+    "strict": ["error", "global"],
+    "function-paren-newline": ["error", "consistent"]
```

Comment:
We are trying to keep the list of formatting-related settings sorted, could you please move this new line in between `"func-call-spacing"` and `"indent"` lines?